pynear/tests/integration/test_cli.py

```diff
@@ -0,0 +1,197 @@
+import json
+import os
+import random
+
+import delegator
+import pytest
+from retrying import retry
+
+from near.pynear.lib import NearLib
+
+nearlib = NearLib('http://localhost:3030/')
+
+
+@retry(stop_max_attempt_number=5, wait_fixed=1000)
+def check_devnet_health(process):
+    if not process.is_alive:
+        return False
+
+    return nearlib.check_health()
+
+
+@pytest.fixture
+def make_devnet(request):
+    def _make_devnet(base_dir):
+        devnet_exe = os.environ['NEAR_DEVNET_EXE']
+        command = "{devnet_exe} -d {base_dir} --test-block-period 5" \
+            .format(devnet_exe=devnet_exe, base_dir=base_dir)
+        process = delegator.run(command, block=False)
+        request.addfinalizer(process.kill)
+        return check_devnet_health(process)
+
+    return _make_devnet
+
+
+@pytest.fixture(scope='session')
+def hello_wasm_path():
+    cur_dir = os.path.abspath(os.path.dirname(__file__))
+    hello_dir = os.path.join(cur_dir, '../../../tests/hello')
+    command = 'npm install && npm run build'
+    process = delegator.run(command, cwd=hello_dir)
+    assert process.return_code == 0, process.err
+    return os.path.join(hello_dir, '../hello.wasm')
+
+
+class Helpers(object):
+    @staticmethod
+    def run_command(command):
+        command = "python -m near.pynear.cli {}".format(command)
+        process = delegator.run(command)
+        assert process.return_code == 0, process.err
+        return process.out
+
+    @classmethod
+    def get_latest_beacon_block(cls):
+        command = 'view_latest_beacon_block'
+        out = cls.run_command(command)
+        return json.loads(out)
+
+    @classmethod
+    def view_account(cls, account_name=None):
+        command = 'view_account'
+        if account_name is not None:
+            command = "{} --account {}".format(command, account_name)
+
+        out = cls.run_command(command)
+        return json.loads(out)
+
+    @classmethod
+    def get_latest_shard_block(cls):
+        command = 'view_latest_shard_block'
+        out = cls.run_command(command)
+        return json.loads(out)
+
+    @classmethod
+    def create_account(cls, account_id):
+        command = "create_account {} 10".format(account_id)
+        cls.run_command(command)
+
+        @retry(stop_max_attempt_number=5, wait_fixed=1000)
+        def _wait_for_account():
+            return cls.view_account(account_id)
+
+        return _wait_for_account()
+
+    @classmethod
+    def deploy_contract(cls, wasm_path):
+        buster = random.randint(0, 10000)
```

Comment:
Fix the seed somewhere.

Reply:
?

Reply:
The seed that random uses should be a constant that is explicitly set, to make tests deterministic and reproducible.